content/log/parsing.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Parsing
+kind: Documentation
+autotocdepth: 2
+hideguides: true
+customnav: lognav
+---
+<div class="alert alert-info">
+Datadog's log management solution is is currently in private beta. If you would like to apply to it, please fill out <a href="https://www.datadoghq.com/log-management/">this form</a>.
+</div>
+
+## Overview 
+
+The Grok syntax provides an easier way to parse logs than pure regular expressions. 
+The main usage of the Grok Parser is to extract attributes from semi-structured text messages.
+
+Grok comes with a lot of reusable patterns to parse integers, ip addresses, hostnames, etc...
+
+Parsing rules can be written with the `%{MATCHER:EXTRACT:FILTER}` syntax:
+
+* **Matcher**: rule (possibly a reference to another token rule) that describes what to expect (number, word, notSpace,...)
+
+* **Extract** (optional): an identifier representing the capture destination for the piece of text matched by the MATCHER.
+
+* **Filter** (optional): a post-processor of the match to transform it
+
+Example for this classic unstructured log:  
+```
+user:john connected on 11/08/2017
```

Comment:
Let's use `john connected on 11/08/2017` everywhere except on the key value example.
Need to update the screenshot as well